src/chopper_count.cpp

```diff
@@ -15,29 +16,50 @@ void initialize_argument_parser(seqan3::argument_parser & parser, chopper::count
     parser.info.short_description = "Count all kmers of each file in a directory.";
     parser.info.version = "1.0.0";
 
-    parser.add_option(config.data_file, 'f', "data_file", "Give me a filename to a seqinfo file.", seqan3::option_spec::required);
-    parser.add_option(config.output_filename, 'o', "outfile", "Filename where the counts should be written to.",
+    parser.add_option(config.data_file,
+                      'f', "data_file",
+                      "Give me a filename to a seqinfo file.",
                       seqan3::option_spec::required);
-    parser.add_option(config.hll_dir, 'd', "hll-dir",
-                      "A directory path where the HyperLogLog sketches will be stored in files. "
-                      "When this is not given, the sketches will not be stored in files.",
-                      seqan3::option_spec::standard,
-                      seqan3::output_directory_validator{});
-    parser.add_option(config.column_index_to_cluster, 'c', "column-index", "The column index by which to cluster.");
-    parser.add_option(config.num_threads, 't', "threads", "Number of threads.");
-    parser.add_option(config.k, 'k', "kmer-size", "The kmer size to count minimisers.");
-    parser.add_option(config.w, 'w', "window-size", "The window size for minimisers.");
-    parser.add_option(config.sketch_bits, 's', "sketch-bits",
+
+    parser.add_option(config.output_prefix,
+                      'o', "output-prefix",
+                      "chopper count creates a `[PREFIX].weights` file that contains an estimated weight for each user "
+                      "bin which is used in chopper layout. Additionally a directory [PREFIX].sketches is created and "
+                      "one `.sketch` file per user bin is computed. The sketch files improve the layout further.",
```

Comment:
I think this needs some re-phrasing (state what is expected input) and an example.
It's not immediately clear what the expected input is.

from `my/path/prefix`, it will create `my/path/prefix.weights` and `my/path/prefix.sketches/*.sketch`

What happens if the path ends in `/`?
Would this be a case for some combined filesystem/regex validator? (as follow-up)

Reply:
I'll clean up the command line afterwards and try to keep this in mind

Reply:
Follow-up